ossec-hids

```diff
@@ -0,0 +1,1444 @@
+
```

Comment:
I think this file was mistakenly added. You should be able to remove it with `git rm ossec-hids && git commit`

Then a `git push` should remove it from the pull request.